setup_ts.py

```diff
@@ -144,6 +144,7 @@ def compile_ts():
     # todo: I don't think these should be here
     shutil.rmtree('ts/nni_manager/dist/config', ignore_errors=True)
     shutil.copytree('ts/nni_manager/config', 'ts/nni_manager/dist/config')
+    shutil.copy('ts/nni_manager/package.json', 'ts/nni_manager/dist/package.json')
```

Comment:
I don't think this should be here. Maybe we can do in `copy_nni_node()`.
Web UI and NAS UI does not copy `package.json` to there compiling output.
And in release mode this `package.json` is not used directly. The version string needs modification.

Reply:
`copy_nni_node()` already has copied `package.json`, should we do this in `symlink_nni_node()`? or use `copy_nni_node()` for all situation in `build()`?

Reply:
Oh, so the problem is node.js only accepts regular file package.json, not symlinked package.json?

Reply:
the problem is `__dirname` is the original location of the file, not symlinked location.